app/handlers/analysis.py

```
@@ -18,4 +26,76 @@ class AnalysisHandler(tornado.web.RequestHandler):
     @apiError error_message The error message to display.
     """
     def post(self):
+        identifier = self.request.body_arguments["identifier"]
```

Do we need to do error handling (i.e. checking if "identifier" exists)?

---

We were ignoring errors for now I believe.

---

Agreed, definitely something that needs to be done before public release. Should create an error handling branch after MVP is complete.